dozer-log-python/src/mapper.rs

```diff
@@ -76,13 +79,34 @@ fn map_value(value: Field, py: Python) -> PyResult<Py<PyAny>> {
         Field::Decimal(v) => Ok(v.to_string().to_object(py)),
         Field::Timestamp(v) => Ok(v.to_string().to_object(py)),
         Field::Date(v) => Ok(v.to_string().to_object(py)),
-        Field::Bson(v) => Ok(v.to_object(py)),
+        Field::Json(v) => map_json_py(v, py),
         Field::Point(v) => map_point(v, py),
         Field::Duration(v) => Ok(v.to_string().to_object(py)),
         Field::Null => Ok(py.None()),
     }
 }
 
+fn map_json_py(val: JsonValue, py: Python) -> PyResult<Py<PyAny>> {
+    match val {
+        JsonValue::Null => Ok(py.None()),
+        JsonValue::Bool(b) => Ok(b.to_object(py)),
+        JsonValue::Number(n) => Ok(n.to_object(py)),
+        JsonValue::String(s) => Ok(s.to_object(py)),
+        JsonValue::Array(a) => Ok(PyList::new(
+            py,
+            a.into_iter().map(|val| map_json_py(val, py).unwrap()),
```

Comment:
Caught one more `unwrap`!

Reply:
changed like how we handle JsonValue::Object type